File: applications/admin-web/src/app/shared/services/teacher/models/teacher-dto.ts
import { EntityDto } from '@abp/ng.core';

export class TeacherDto extends EntityDto<string>{
  name: string;
  birthday: Date;
  phoneNumber: string;
  email: string;
  address: string;
  degree: string;
  faculty: string;
  position: string;

  constructor(initial: Partial<TeacherDto> = {}) {
    super(initial);
  }
}

File: servicces/src/doan.ProjectManagement.Application.Contracts/Teachers/Dto/TeacherDto.cs
﻿using Abp.Application.Services.Dto;
using System;

namespace doan.ProjectManagement.Teachers.Dto
{
    public class TeacherDto : FullAuditedEntityDto<Guid>
    {
        public string Name { get; set; }
        public DateTime? Birthday { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public string Degree { get; set; }
        public string Faculty { get; set; }
        public string Position { get; set; }
    }
}
